--- liborcus-0.5.1/src/Makefile.am.orig	2013-03-25 01:18:18.000000000 +0100
+++ liborcus-0.5.1/src/Makefile.am	2013-04-26 15:24:37.638182274 +0200
@@ -36,7 +36,8 @@
 
 orcus_xml_dump_LDADD = \
 	parser/liborcus-parser-@ORCUS_API_VERSION@.la \
-	liborcus/liborcus-@ORCUS_API_VERSION@.la
+	liborcus/liborcus-@ORCUS_API_VERSION@.la \
+	-lpthread
 
 orcus_xml_dump_CPPFLAGS = -I$(top_builddir)/lib/liborcus/liborcus.la $(COMMON_CPPFLAGS)
 
@@ -110,7 +111,8 @@
 orcus_xml_LDADD = \
 	liborcus/liborcus-@ORCUS_API_VERSION@.la \
 	parser/liborcus-parser-@ORCUS_API_VERSION@.la \
-	spreadsheet/liborcus-spreadsheet-model-@ORCUS_API_VERSION@.la
+	spreadsheet/liborcus-spreadsheet-model-@ORCUS_API_VERSION@.la \
+	-lpthread
 
 orcus_xml_CPPFLAGS = -I$(top_builddir)/lib/liborcus/liborcus.la $(COMMON_CPPFLAGS)
 
--- liborcus_0.3.0/src/spreadsheet/Makefile.am.orig	2012-11-28 13:21:10.000000000 +0100
+++ liborcus_0.3.0/src/spreadsheet/Makefile.am	2013-04-01 07:28:42.121989280 +0200
@@ -17,6 +17,6 @@
 
 liborcus_spreadsheet_model_@ORCUS_API_VERSION@_la_CXXFLAGS = $(LIBIXION_CFLAGS)
 liborcus_spreadsheet_model_@ORCUS_API_VERSION@_la_LDFLAGS = -no-undefined
-liborcus_spreadsheet_model_@ORCUS_API_VERSION@_la_LIBADD = $(LIBIXION_LIBS)
+liborcus_spreadsheet_model_@ORCUS_API_VERSION@_la_LIBADD = $(LIBIXION_LIBS) $(top_builddir)/src/liborcus/liborcus-@ORCUS_API_VERSION@.la
 
 endif
--- liborcus-0.5.1/src/parser/Makefile.am.orig	2013-04-12 16:25:03.000000000 +0200
+++ liborcus-0.5.1/src/parser/Makefile.am	2013-04-26 15:43:10.978135549 +0200
@@ -20,6 +20,7 @@
 
 liborcus_parser_@ORCUS_API_VERSION@_la_LDFLAGS = -no-undefined
 liborcus_parser_@ORCUS_API_VERSION@_la_LIBADD = \
+	$(BOOST_SYSTEM_LIB) \
 	 $(ZLIB_LIBS)
 
 EXTRA_PROGRAMS = \
--- liborcus-0.5.1/src/liborcus/Makefile.am.orig	2013-04-12 16:24:50.000000000 +0200
+++ liborcus-0.5.1/src/liborcus/Makefile.am	2013-04-26 16:53:18.567958975 +0200
@@ -175,6 +175,7 @@
 	$(BOOST_SYSTEM_CFLAGS) $(ZLIB_CFLAGS)
 liborcus_@ORCUS_API_VERSION@_la_LDFLAGS = -no-undefined
 liborcus_@ORCUS_API_VERSION@_la_LIBADD = \
+	$(top_builddir)/src/parser/liborcus-parser-@ORCUS_API_VERSION@.la \
 	 $(BOOST_SYSTEM_LIBS) $(BOOST_SYSTEM_LIB) $(ZLIB_LIBS)
 
 # liborcus-test-xml-map-tree
