--- liborcus-0.18.1/configure.ac.orig	2024-08-29 18:05:17.828527833 +0200
+++ liborcus-0.18.1/configure.ac	2024-08-29 18:11:03.589288016 +0200
@@ -109,10 +109,9 @@ AC_ARG_ENABLE(debug,
         [enable_debug=no]
 )
 AS_IF([test "x$enable_debug" != "xno"], [
-        CXXFLAGS="$CXXFLAGS -g -O0"
+        CXXFLAGS="$CXXFLAGS -g"
         enable_debug_utils=yes
 ], [
-        CXXFLAGS="$CXXFLAGS -O2"
         AC_DEFINE([NDEBUG], [], [Disable debugging information])
 ])
 
