--- liborcus-0.16.1/configure.ac.orig	2020-10-31 08:29:30.443076469 +0100
+++ liborcus-0.16.1/configure.ac	2020-10-31 08:30:02.636235397 +0100
@@ -112,10 +112,9 @@
         [enable_debug=no]
 )
 AS_IF([test "x$enable_debug" != "xno"], [
-        CXXFLAGS="$CXXFLAGS -g -O0 -DMDDS_ASSERT_STRING_MAP"
+        CXXFLAGS="$CXXFLAGS -g -DMDDS_ASSERT_STRING_MAP"
         enable_debug_utils=yes
 ], [
-        CXXFLAGS="$CXXFLAGS -O2"
         AC_DEFINE([NDEBUG], [], [Disable debugging information])
 ])
 
